integreat_cms/core/settings.py

```diff
@@ -68,6 +68,14 @@
 #: URL to the Integreat wiki
 WIKI_URL = os.environ.get("INTEGREAT_CMS_WIKI_URL", "https://wiki.integreat-app.de")
 
+#: Platform Name, used for app users and social media headers
+PLATFORM_NAME = os.environ.get("INTEGREAT_PLATFORM_NAME", "Integreat")
```

Comment:
I think it makes sense to combine this setting with the existing `BRANDING` value to avoid having to maintain both settings.
I opened #2434 for this matter, and in case that PR gets merged first, I'd suggest to remove the `PLATFORM_NAME` setting and use `BRANDING_TITLE` instead.

Reply:
```suggestion
```
The PR is now merged, so you could rebase this one and replace `PLATFORM_NAME` with `BRANDING_TITLE` everywhere.